sklearn/cluster/_hdbscan/_linkage.pyx

```diff
@@ -45,6 +45,8 @@ cpdef cnp.ndarray[MST_edge_t, ndim=1, mode='c'] mst_from_mutual_reachability(
         represented as a collecteion of edges.
     """
     cdef:
+        # Note: we utilize ndarray's over memory-views to make use of numpy
+        # binary indexing and sub-selection below.
```

Comment:
Thank you for this comment, this answers one of @Vincent-Maladiere's questions. :+1: